packages/api-client/src/hooks/useDarkModeState.ts

```diff
@@ -1,73 +1,98 @@
-import { ref, watch } from 'vue'
+import { onMounted, onUnmounted, ref, watch } from 'vue'
 
-// State
-const isDark = ref<boolean>(false)
+// State - null to indicate system preference
```

Comment:
Probably don't need this comment anymore

Reply:
```suggestion

```